tests/phpunit.xml

```diff
@@ -7,4 +7,10 @@
     <testsuite>
         <directory>./</directory>
     </testsuite>
+    <php>
+        <var name="DB_DSN" value="sqlite::memory:" />
```

Comment:
I'm guessing "DSN" stands for "Database Server Name?"  Please rename to `DB_SERVER` or something a bit more descriptive.